app/tests/serving_tests/test_views.py

```diff
@@ -76,12 +76,18 @@ def test_imageset_annotationset_download(
 
 
 @pytest.mark.django_db
-def test_image_response(client):
+@pytest.mark.parametrize("internal", (True, False))
+def test_image_response(client, internal):
     image_file = ImageFileFactory()
     user = UserFactory()
 
+    if internal:
+        data = {"internal": True}
+    else:
+        data = {}
+
```

Comment:
A bit nitpicky, but you could save these lines by parametrizing with the `data` dict itself.

Reply:
Nope, the test will fail that way. We don't actually inspect the value of the `internal` query parameter, RFC2396 doesn't define how booleans should appear in uris, so in the view we only test for the presence of this key.

Reply:
This could be a bug, I guess, but I didn't want to do in to how we would define truthy values here.

Reply:
Ah sorry, I was a bit too quick with my comment. I agree that this is the best way to go.